styling/mapbox-gl-geocoder.css

```diff
@@ -0,0 +1,233 @@
+Basics .mapboxgl-ctrl-geocoder,
```

Comment:
Just wondering, is "Basics" supposed to be there?

Reply:
I copied that from their own CSS I'm not sure what it does but I just kept it in since it was in the original